python/tests/test_highlevel.py

```diff
@@ -541,6 +561,149 @@ def test_samples(self):
             ]
         assert total == ts.num_samples
 
+    def test_samples_time(self):
+        times = [0, 0.1, 1 / 3, 1 / 4, 5 / 7]
+        ts = self.get_tree_sequence(num_demes=2, n=20, times=times)
+        for time in times:
+            assert np.array_equal(get_samples(ts, time=time), ts.samples(time=time))
+            for population in (None, 0):
+                assert np.array_equal(
+                    get_samples(ts, time=time, population=population),
+                    ts.samples(time=time, population=population),
+                )
+
```

Comment:
These are good candidates for parametrize:
```python
@pytest.mark.parametrize("time_interval", [
     [0, 0.1],
￼            (0, 1 / 3),
￼            np.array([1 / 4, 2 / 3]),
￼            (0.345, 5 / 7),
￼            (-1, 1),
￼        ])
def test_samples_time_interval(self, time_interval):
     # body of the existing loop
```
can do the same thing with ``test_samples_time`` above.

This approach is better because it tells you *which* of the cases failed when it fails.

Reply:
oh, cool! didn't know about that. will update.